Nukes some of the overpowered imports, rebalances some of the weapons (ESPECIALLY bogseo)

F-e-r-n opened the pull request:
## About The Pull Request
Removes some of the imports that invalidate science progression like the surgical tools and anomaly core modules

Nerfs the Bogseo and changes it into a slower firing, double tap burst weapon. It's still strong just not *click* ur ded.

Rebalances import armor and weapons a tad, sacrificial armor was taken out as I genuinely do not know how it could be balanced without being changed conceptually. A nerfed version can still be spawned by admins.

Makes some of the import ammos not have ridiculous negative wound chances, straight up deletes the negative wound chance on some of the weaker bullets. Seriously what are these, nerf darts?

Rebalances the apds (tungsten) slug to be a sidegrade rather than basically a straight upgrade
## Why It's Good For The Game
European players should not be abused by the bogseo. Science should be a department that actually contributes to the round.
## Changelog
:cl: Syndie al gaib
del: Removes most science-locked things from the imports menu
balance: Rebalances the Bogseo
balance: Plays around with some of the imports' numbers
del: Removes some of the more risque modular laser lines
add: Adds a few references that I thought were funny to the modular lasers
:cl:

iamcooldan commented:
this is all looking fairly good to me, (and thats my bias that the import armors look ugly) but I will have to say it might be good to look at blueshield loadouts for the trappiste ammo change, since they do have the potential of a trappiste revolver and this might nerf their effectiveness for what is already a fairly meh pistol.

KnigTheThrasher commented:
looking at the code you didnt even bother changing the comments on some stuff and changed what weapon say for no reason

F-e-r-n commented:
> looking at the code you didnt even bother changing the comments on some stuff and changed what weapon say for no reason

1. this isn't complete, im still finding stuff its just ready for tm
2. the majority of what i removed were innuendos and/or played into the "sexy oriental woman" archetype, which is not something we want on our server.

KnigTheThrasher commented:
> > looking at the code you didnt even bother changing the comments on some stuff and changed what weapon say for no reason
> 
>     1. this isn't complete, im still finding stuff its just ready for tm
> 
>     2. the majority of what i removed were innuendos and/or played into the "sexy oriental woman" archetype, which is not something we want on our server.

put it in the changelog then?

F-e-r-n commented:
> > > looking at the code you didnt even bother changing the comments on some stuff and changed what weapon say for no reason
> > 
> > 
> > ```
> > 1. this isn't complete, im still finding stuff its just ready for tm
> > 
> > 2. the majority of what i removed were innuendos and/or played into the "sexy oriental woman" archetype, which is not something we want on our server.
> > ```
> 
> put it in the changelog then?

fiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine

F-e-r-n commented:
> this is all looking fairly good to me, (and thats my bias that the import armors look ugly) but I will have to say it might be good to look at blueshield loadouts for the trappiste ammo change, since they do have the potential of a trappiste revolver and this might nerf their effectiveness for what is already a fairly meh pistol.

takbok was already balanced, its already a rather strong gun and hollowpoint turned it into a wannabe syndie revolver with its crazy wounding.

Loiosh42 commented:
Alright, whole bunch of stuff I feel needs changing before this should be merged.

APDS slugs: Are you CERTAIN you want to increase the damage on these? They do pierce mobs, this feels more like hiding a buff than nerfing it. AP value being a bit lower is somewhat reasonable, but 40 damage mob piercing with AP feels pretty absurd to me. You don't usually fight a lot of armor, they were purposefully weaker against soft targets.

Advanced med tools, the e-guns, razorwire, modsuit modules... fair enough.

Why the hell is sacrificial armor getting nerfed to hell AND being removed to admin only???????
Nerfing it and leaving it is reasonable.
Removing it to admin spawn is reasonable.
Nerfing it AND removing to admin spawn makes NO sense whatsoever to me.

I think the bare wound bonus on the .585 could be a little higher, but fair enough.

Bogeso needs a nerf, yes, I don't think it needs a spread increase however. You do realize you have increased the wound chance of the Bogeso from where it was by 50 units? ( Rest of the changes seem fine to me.

Why are you nerfing wounding on the Wylom rounds?
Why are you removing the negative wound bonus on the lanca?

Line changes for the lasers seem fine.

F-e-r-n commented:
> APDS slugs: Are you CERTAIN you want to increase the damage on these? They do pierce mobs, this feels more like a buff than a sidegrade/nerf. AP value being a bit lower is somewhat reasonable, but 40 damage mob piercing with AP feels pretty absurd to me. Most things you shoot don't have that much armor, they were purposefully weaker against soft targets.

Eh this one is fair, I was moreso bringing it in line with standard damage-wise at 50% armor+, piercing is funny when it happens and rarely affects the outcome of a combat situation, I might nerf it overall and make it technically a straight downgrade against single targets, but to me it reads as a fun once in a while 'collateral!' situation. If I were to do anything I might just make it lose damage and ap on ricochet.

> Why the hell is sacrificial armor getting nerfed to hell AND being removed to admin only??????? Nerfing it and leaving it is reasonable. Removing it to admin spawn is reasonable. Nerfing it AND removing to admin spawn makes NO sense whatsoever to me.
ORIGINALLY I was going to keep it in, but honestly everything I tried didn't feel quite right for crew balance, so I removed it. I left it in the code itself with a bit of a rebalance (it was genuinely ridiculous prior) incase it gets used again in the future. It is a cool concept I just don't believe it's good as an import buy.
> I don't think the other armor needs the extra nerfs to begin with, especially with the price hike coming soon.
They definitely do, they were some of the best all-round gear available. I might retouch them a little bit but yknow
> I think the bare wound bonus on the .585 could be a little higher, but fair enough.
> 
> Bogeso needs a nerf, yes, I don't think it needs a spread increase however. You do realize you have increased the wound chance of the Bogeso from where it was by 50 units? ( Rest of the changes seem fine to me.
Yes, It's supposed to be a FAT SMG now instead of the fucking ridiculous mag spitter it used to be. I changed wound bonus to reflect this. Honestly, the spread just felt better to me and allowed me to decrease its viable range a little bit.
> Why are you nerfing wounding on the Wylom rounds? Why are you removing the negative wound bonus on the lanca?
The numbers themselves were kinda ridiculous, and the negative wound bonus on the lanca was extra ridiculous (seriously it lost wounding on naked people compared to clothed people). Again, might touch these some more if they prove to be too much or too little.

Loiosh42 commented:
![image](https://github.com/user-attachments/assets/2e612ad0-5d7d-4950-b8d2-8f71ff1e642e)
Saboted is the correct word there.

RandomDudeFromTheRim commented:
To be honest, I think it'd be nice to keep the anomaly modules BUT without the cores. Because there might be those moments when science has the cores, but does not have the mats or the research to make the module (most likely the former)
